### DISCUSSION TITLE
Acess to Copilot / Acesso ao Copilot

### DISCUSSION THREAD

**Discussion by ghost** [community/community#110110] [GitHub Education]
### Select Topic Area

Question

### Body

Hello, I'm trying to request access to github copilot because I'm a student at a public university, but the university I study at doesn't provide an institutional email address, an acquaintance told me that it's possible to make this request with my personal email as long as I have a declaration, but I haven't found this option while doing my research for the request, would anyone know if they could help me?
-------------------
Olá estou tentando solicitar acesso ao github copilot por ser estudante de Universidade pública, porém a Universidade em que estudo não fornece email instutucional, um conhecido me informou que é possível fazer essa solicitação com o email pessoal desde que eu possua uma declaração, porem não encontrei essa opção enquanto faco minhas pesquisas para solicitação, alguém saberia conseguiria me ajudar ?

**Comment by UpekHansaja** [comment]
Hi there,
you can apply to [GitHub Student Developer Pack ](https://education.github.com/pack) and with that you will obtain free access to GitHub Copilot.
It's totally fine with applying using your personal email address but make sure the email address you are willing to apply & the email address provided for your university are same.


Thank you..

**Comment by ghost** [comment]
They won't let me, I've tried contacting support but to no avail :(
What I'm doing is trying to enter my email address and then inform my institution, but this always happens.
-----------------------------------------------------------------------------------------------------------
Não permitem tentei entrar em contato com o suporte mas sem sucesso :(
O que estou fazendo é tentando inserir o meu email e então informar a minha instituição mas sempre da isso.
![Captura de Tela 2024-03-02 às 21 26 46](https://github.com/community/community/assets/89662293/dfe90bf4-2d0f-4013-8c0b-0edc95a425d9)
